doc/modules/gaussian_process.rst

```diff
@@ -570,7 +570,7 @@ It is parameterized by a length-scale parameter :math:`l>0` and a periodicity pa
 The kernel is given by:
 
 .. math::
-   k(x_i, x_j) = \text{exp}\left(-2 \left(\text{sin}(\pi / p * d(x_i, x_j)) / l\right)^2\right)
+   k(x_i, x_j) = \text{exp}\left(- \frac{ 2\sin^2(\pi |x_i - x_j| / p}{ l^ 2} \right)
```

Comment:
here as well

Reply:
As above, but also, as a mathematician, I don't like using * in formulas because it is also used for other operations such as convolutions.
\frac is added to improve readability.

Also replaced the metric function $d(., .)$ with $|x_i - x_j|$, it doesn't seem like we could use any metric here (at least don't see this in the references and I assume it's also not implemented)

Reply:
the sign is missing a closing paranthesis now